docs/components/board/_index.md

```diff
@@ -0,0 +1,1534 @@
+- `timeout` [(Optional[float])](https://docs.python.org/library/typing.html#typing.Optional): An option to set how long to wait (in seconds) before calling a time-out and closing the underlying RPC call.
+
+**Returns:**
+
+- `rolling_avg` [(int)](https://docs.python.org/3/library/functions.html#int) The [RollingAverage](https://pkg.go.dev/go.viam.com/rdk/utils#RollingAverage) of the time in nanoseconds between two successive low signals (pulse width) recorded by [`Tick()`](#tick), computed over a window of size `10`.
+
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/components/board/index.html#viam.components.board.Board.DigitalInterrupt.value).
+
+```python
+my_board = Board.from_robot(robot=robot, name="my_board")
+
+# Get the DigitalInterrupt "my_example_digital_interrupt".
+interrupt = await my_board.digital_interrupt_by_name(name="my_example_digital_interrupt")
+
+# Get the rolling average of the pulse width across each time the DigitalInterrupt is interrupted with a tick.
+rolling_avg = await interrupt.value()
+```
+
+{{% /tab %}}
+{{% tab name="Go" %}}
+
+**Parameters:**
+
+- `Context` [(Context)](https://pkg.go.dev/context): A Context carries a deadline, a cancellation signal, and other values across API boundaries.
+- `extra` [(map[string]interface{})](https://pkg.go.dev/google.golang.org/protobuf/types/known/structpb): Extra options to pass to the underlying RPC call.
+
+**Returns:**
+
+- `rolling_avg`[(int64)](https://pkg.go.dev/builtin#int64): The [RollingAverage](https://pkg.go.dev/go.viam.com/rdk/utils#RollingAverage) of the time in nanoseconds between two successive low signals (pulse width) recorded by [`Tick()`](#tick), computed over a window of size `10`.
+- `error` [(error)](https://pkg.go.dev/builtin#error): An error, if one occurred.
+
+For more information, see the [Go SDK Docs](https://pkg.go.dev/go.viam.com/rdk/components/board#ServoDigitalInterrupt).
+
+```go
+myBoard, err := board.FromRobot(robot, "my_board")
+
+// Get the DigitalInterrupt "my_example_digital_interrupt".
+interrupt, ok := myBoard.DigitalInterruptByName("my_example_digital_interrupt")
+
+// Get the rolling average of the pulse width across each time the DigitalInterrupt is interrupted with a tick.
+rolling_avg, err := interrupt.Value(context.Background(), nil)
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+{{% /tab %}}
+{{< /tabs >}}
+
+### Tick
+
+Record an interrupt and notify any [channels](https://go.dev/tour/concurrency/2) that have been added with [AddCallback()](#addcallback).
+
+{{% alert title="Caution" color="caution" %}}
+You should only need to integrate this method into your application code for testing purposes, as the handling of `Tick()` should be automated once the interrupt is configured.
+
+Calling this method is not yet fully implemented with the Viam Python SDK.
```

Comment:
@njooma or @penguinland can either of you confirm this? it's documented in python SDK docs but I found in the python sdk repo that it would just raise a not-implemented error, so I commented out the docs I wrote for python. Same for AddCallback() and AddPostProcessor() method, also on digital interrupts

Reply:
I believe it. `Tick` is only intended to be called from unit tests, to simulate a new signal coming in on hardware. I don't understand why you'd ever want to call it through an SDK from a different language (why not write your tests in the same language you wrote your component in?).